packages/cozy-harvest/src/Harvest.spec.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Harvest from './Harvest'
+import renderer from 'react-test-renderer'
```

Comment:
We are using Enzyme for shallow rendering, see https://github.com/cozy/cozy-store/blob/master/test/components/appsLoading.spec.js#L6 for example